optimum/commands/export/onnx.py

```diff
@@ -217,6 +217,11 @@ def parse_args_onnx(parser):
         default=DEFAULT_DUMMY_SHAPES["nb_points_per_image"],
         help="For Segment Anything. It corresponds to the number of points per segmentation masks.",
     )
+    optional_group.add_argument(
+        "--legacy",
+        action="store_true",
+        help=("Export decoder only models in two (without + with past) model as a single ONNX file."),
```

Comment:
```suggestion
        help="Export decoder only models in two (without + with past) model as a single ONNX file.",
```
QQ: the `--legacy` flag should export two ONNX files no?

Reply:
It actually export 3 ONNX files (without past, with past, merged decoder), I updated the description so that it's clearer